app/controllers/inherited_proofing_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'inherited_proofing/mhv/inherited_proofing_verifier'
+require 'inherited_proofing/logingov/service'
+
+class InheritedProofingController < ApplicationController
+  skip_before_action :verify_authenticity_token, :authenticate
+
+  def auth
+    load_user
```

Comment:
Since we shouldn't skip `authenticate` for this route, we don't need to manually `load_user`, and can assume `@current_user` is defined